### ISSUE TITLE
Plot recipe for rbf is broken

### ISSUE THREAD

**Issue by juliohm** [baggepinnen/BasisFunctionExpansions.jl#5]
Copying/pasting the example in the docs:

```julia
N   = 200
v   = linspace(0,10,N)
y   = 0.1*(v-2).*(v-7) + 0.2randn(N)
rbf = UniformRBFE(v, 5, normalize = true)
bfa = BasisFunctionApproximation(y,v,rbf)

scatter(v,y,lab="Signal",c=:orange, subplot=1, xlabel="\$v\$", size=(600,300))
plot!(rbf) # this is broken
plot!(v,bfa(v),lab="Reconstruction",c=:blue,linewidth=2)

UndefVarError: HSV not defined
```

**Comment by baggepinnen** [comment]
Thanks! Will update this in two weeks when I'm back from vacation!

**Comment by baggepinnen** [comment]
Fixed in 75db41d
